packages/material-bottom-tabs/src/views/MaterialBottomTabView.tsx

```diff
@@ -177,14 +175,21 @@ function MaterialBottomTabViewInner({
 
         return null;
       }}
-      getLabelText={({ route }: Scene) => {
+      renderLabel={({ route, focused, color }) => {
         const { options } = descriptors[route.key];
 
-        return options.tabBarLabel !== undefined
-          ? options.tabBarLabel
-          : options.title !== undefined
-          ? options.title
-          : (route as Route<string>).name;
+        const label =
+          options.tabBarLabel !== undefined
+            ? options.tabBarLabel
+            : options.title !== undefined
+            ? options.title
+            : (route as Route<string>).name;
+
+        if (typeof label === 'string') {
+          return label;
```

Comment:
This shouldn't return a string. It should use `renderLabel` only if the label is a function, otherwise use the `getLabelText` prop. Without this the styling for the label will be incorrect when a string is used.

Reply:
Using `getLabelText` will preserve the previous behavior (e.g. use React Native Paper's theming system)